packages/core-components/api-report.md

```diff
@@ -921,15 +921,11 @@ export const sidebarConfig: {
   mobileSidebarHeight: number;
 };
 
+// Warning: (ae-forgotten-export) The symbol "SidebarContextType" needs to be exported by the entry point index.d.ts
+//
 // @public @deprecated
 export const SidebarContext: React_2.Context<SidebarContextType>;
 
-// @public @deprecated
-export type SidebarContextType = {
-  isOpen: boolean;
-  setOpen: (open: boolean) => void;
-};
```

Comment:
We don't want to remove this as part of this cleanup. Applies to both this one and the `SidebarPinStateContext`/`SidebarPinStateContextType` below.